main/ZgatewayBT.ino

```diff
@@ -1029,7 +1035,7 @@ void launchBTDiscovery() {
       String macWOdots = String(p->macAdr);
       macWOdots.replace(":", "");
       Log.trace(F("Launching discovery of %s" CR), p->macAdr);
-      if (p->sensorModel == HHCCJCY01HHCC) MiFloraDiscovery((char*)macWOdots.c_str(), "HHCCJCY01HHCC");
```

Comment:
I'm not sure why the miflora doesn't need the (char *) here, should it be removed for all discovery calls?

Reply:
Yes, that does seem odd doesn't it. The reason was I simply did a quick compile check with just one and forgot to go back and change the rest. I'll remove the others and push the change.

Reply:
Done